install.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
Not needed for this tutorial. Keep it if you want to, but remove this script from this repository.

Reply:
@MaximeAubanel

Reply:
Done